src/ipc/dbus/DBusIPCProxy.h

```diff
@@ -160,6 +160,8 @@ class DBusIPCProxy : public IPCProxyBase<InterfaceType>, protected DBusIPCProxyB
 
 public:
     typedef const char *MemberIDType;
```

Comment:
Nitpick: mixture between typedef and using declarations - if you are at it, could you switch to "using"?

Reply:
done with next commits